tools/test-js-with-node

```diff
@@ -191,8 +191,22 @@ parser = argparse.ArgumentParser(USAGE)
 parser.add_argument("--coverage", action="store_true", help="Get coverage report")
 add_provision_check_override_param(parser)
 parser.add_argument("args", nargs=argparse.REMAINDER)
+parser.add_argument(
+    "--parallel",
+    dest="parallel",
+    action="store",
+    type=int,
+    default=os.cpu_count(),
+    help="Specify the number of processes to run the "
+    "tests in. Default is the number of logical CPUs",
+)
 options = parser.parse_args()
 individual_files = options.args
+parallel = options.parallel
+
+if options.coverage and parallel > 1:
+    parallel = 1
+    print(BOLDRED + "You cannot use --coverage with parallel tests\n" + ENDC)
```

Comment:
I would also add something like "Running in serial mode" at the end of that statement.